rocketpy/simulation/flight.py

```diff
@@ -3045,41 +3045,20 @@ def export_pressures(self, file_name, time_step):
         ------
         None
         """
-
         time_points = np.arange(0, self.t_final, time_step)
-
-        # Create the file
-        file = open(file_name, "w")
-
-        if len(self.rocket.parachutes) == 0:
-            pressure = self.env.pressure(self.z(time_points))
-            for i in range(0, time_points.size, 1):
-                file.write("{:f}, {:.5f}\n".format(time_points[i], pressure[i]))
-
-        else:
-            for parachute in self.rocket.parachutes:
-                for i in range(0, time_points.size, 1):
-                    pCl = Function(
-                        parachute.clean_pressure_signal,
-                        "Time (s)",
-                        "Pressure - Without Noise (Pa)",
-                        "linear",
-                    )(time_points[i])
-                    pNs = Function(
-                        parachute.noisy_pressure_signal,
-                        "Time (s)",
-                        "Pressure - With Noise (Pa)",
-                        "linear",
-                    )(time_points[i])
-                    file.write(
-                        "{:f}, {:.5f}, {:.5f}\n".format(time_points[i], pCl, pNs)
-                    )
-                # We need to save only 1 parachute data
-                pass
-
-        file.close()
-
-        return None
+        with open(file_name, "w", encoding="utf-8") as file:
```

Comment:
I don't know if the encoding matters much here, but maybe not specifying it is the best way forward so that the encoding used is platform-dependent

Reply:
to discuss later:
https://peps.python.org/pep-0686/
https://www.reddit.com/r/Python/comments/2c92sq/debate_enforcing_python_source_encoding_as_utf8/

Reply:
Ok, this is actually the best PEP to understand about the problem: https://peps.python.org/pep-0597/

Summarizing... It actually does not matter that much in this case since we are just saving a text file with numbers separated by comma. Any encoding should work here. However, not specifying the encoding when using `open()` is a bad practice since PEP0597. The 'utf-8' is standard in Python and is usually more flexible than ascii.

To be "platform-dependent", i.e. to write the files using the same encoding as the local machine, it's possible to use encoding="locale", but this only works on python 3.10 and above. See this section for a short description: https://peps.python.org/pep-0597/#for-experienced-users

All in all, this is not a problem for now. I'm expecting that one day in the future we can revisit this issue in the long term future when Python start to enforce the explicit encoding definition everywhere. 

I will revert the change for now, thanks for pointing that out.